reflex-backend-socket.cabal

```diff
@@ -19,52 +19,51 @@ library
   exposed-modules:     Reflex.Backend.Socket
                      , Reflex.Backend.Socket.Connect
                      , Reflex.Backend.Socket.Accept
-  build-depends:       base              >= 4.9    && < 4.13
-                     , binary            >= 0.8    && < 0.11
-                     , bytestring        >= 0.10   && < 0.11
-                     , containers        >= 0.5    && < 0.7
-                     , lens              >= 4.15.4 && < 4.18
-                     , mtl               >= 2.2    && < 2.3
-                     , network           >= 2.6    && < 3.1
-                     , reflex            >= 0.5    && < 0.7
-                     , stm               >= 2.4    && < 2.6
-                     , these             >= 0.7    && < 0.9
-                     , dependent-sum     >= 0.4    && < 0.5
-                     , dependent-map     >= 0.2    && < 0.3
-                     , reflex-basic-host >= 0.1    && < 0.2
-                     , reflex-binary     >= 0.1    && < 0.2
+  build-depends:       base              >= 4.9     && < 4.13
+                     , binary            >= 0.8     && < 0.11
+                     , bytestring        >= 0.10    && < 0.11
+                     , containers        >= 0.5     && < 0.7
+                     , dependent-map     >= 0.2     && < 0.3
+                     , dependent-sum     >= 0.4     && < 0.5
+                     , microlens-th      >= 0.4.2.2 && < 0.5
```

Comment:
If we're pulling in `reflex`, then we are pulling in `lens` transitively anyhow - see [here](https://github.com/reflex-frp/reflex/blob/develop/reflex.cabal#L51).  So adding `microlens` is probably increasing the dependency footprint.

Reply:
Have switched back to `lens`.